CHANGELOG.md

```diff
@@ -12,13 +12,15 @@ The following emojis are used to highlight certain changes:
 * 🛠 - BREAKING CHANGE.  Action is required if you use this functionality.
 * ✨ - Noteworthy change to be aware of.
 
-## [Unreleased]
+## [v0.24.1]
```

Comment:
nit: nemove empty sections from here & create empty `## [Unreleased]` section above this.